lib/check_reqs.js

```diff
@@ -110,7 +110,7 @@ module.exports.get_gradle_wrapper = function () {
     let program_dir;
     // OK, This hack only works on Windows, not on Mac OS or Linux.  We will be deleting this eventually!
     if (module.exports.isWindows()) {
-        const result = execa.sync(path.join(__dirname, 'getASPath.bat'));
+        const result = execa.sync(path.join(__dirname, 'getASPath.bat'), { spawn: true });
```

Comment:
```suggestion
        // "spawn" option enabled for CVE-2024-27980 (Windows) Mitigation
        // See https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2 for more details
        const result = execa.sync(path.join(__dirname, 'getASPath.bat'), { spawn: true });
```

Reply:
Also, I can't find any `spawn` option documented in execa, is this suppose to be `shell` ?

Reply:
I indeed manage to made mistake in the one-liner. Thank you. It's `shell`